### DISCUSSION TITLE
Where does Rabbit store its information?

### DISCUSSION THREAD

**Discussion by j-xella** [VoxelPrismatic/rabbit.nvim#18] [Q&A] [answered]
Let's say I want to move my Rabbit harpoon and oxide lists to another machine - can it be done by copying some files over?

**Comment by VoxelPrismatic** [comment, marked as answer]
On Linux, installed via Lazy:
```
~/.local/share/nvim/lazy/rabbit.nvim/memory
```

<br><br>

For a universal solution, you can simply peek at Harpoon, Reopen, or Oxide:
```lua
:lua =require("rabbit").plugins.harpoon.memory
```
Which returns this for me:
```
/home/priz/.local/share/nvim/lazy/rabbit.nvim/memory/harpoon.rabbit.plugin
```
